kernelci/test.py

```diff
@@ -16,8 +16,11 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
 
 import os
+import re
```

Comment:
What about using a more Pythonic style instead of regex, as we are already importing os ?
For reference, this is how I did in the previous pull request https://github.com/kernelci/kernelci-core/blob/a338de1c8f1fa0703bc56e6f78719af9e37fe3db/kernelci/config/test.py#L265

Reply:
Sure, but I think it would mean doing something like this:
```python
import urllib.parse
import os

COMPRESSION_FORMATS = ['gz', 'bz2', xz']

def get_compression(url):
    path = urllib.parse.spliturl(url).path
    ext = os.path.splitext(path)
    fmt = ext.rpartition('.')[2]
    return fmt if fmt in COMPRESSION_FORMATS else ''
```
So it's a bit more complicated than the regex implementation.

Which one do you prefer?  Or, do you have another approach in mind?

Reply:
ok works for me